owners/src/github.js

```diff
@@ -105,6 +105,8 @@ class Team {
    */
   constructor(id, org, slug) {
     Object.assign(this, {id, org, slug, members: []});
+    const message = 'something something' + 'something';
```

Comment:
The example I meant here is for the case where a newline _is_ required (to avoid line-too-long issues) but you forget to add a space inside the concat

Reply:
(this had a newline but gulp fix removed it)